mnelab/mainwindow.py

```diff
@@ -37,6 +38,7 @@
 __version__ = "0.6.0.dev0"
 
 MAX_RECENT = 6  # maximum number of recent files
+mp.set_start_method("spawn", force=True)  # required for Linux/macOS
```

Comment:
Could you move this line into `__main__.py` (line 84)? I think this doesn't have to be part of `mainwindow.py`.

Reply:
Done in 7587c8b